### PR TITLE
server: provide table disk usage HTTP API for tidb-ctl

### PR DESCRIPTION
**Pull request by tiancaiamao**
Provide HTTP API:
http://127.0.0.1:10080/tables/{db}/{table}/disk-usage

Data take from PD and it's not a precise value, result in MB. 

@shenli @disksing @jackysp

### CODE REVIEW

File: server/region_handler.go [hunk 1]
```diff
@@ -298,7 +307,7 @@ func (rh schemaHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 }
 
 // ServeHTTP handles request of list a table's regions.
-func (rh tableRegionsHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
+func (rh tableHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
```
Comment (shenli) [inline server/region_handler.go]: Update the comment.

**Comment by disksing** [comment]
LGTM.

**Comment by shenli** [comment]
LGTM

**Pull request merged**